src/setup.md

```diff
@@ -6,6 +6,16 @@ and manage different versions of the Rust compiler. Follow the instructions on t
 installed on your machine. For the time being, you'll need Rust nightly when working with wasm:
 
 ```bash
+# Install nightly
+$ rustup install nightly
+
+# Install wasm32-unknown-unknown
+$ rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
wasm-pack actually handles this automatically. You don't kneed to worry about it!

Reply:
Okay, I would remove line 12 and 13 👍